src/pages/economy.hbs

```diff
@@ -0,0 +1,68 @@
+{{#extend "raha/layout/base/layout"}}
+  {{#content "styles" mode="append"}}
+    .wrapper {
```

Comment:
I'm going to rename `.wrapper` to `.economyPage` to avoid any name conflicts with such a general name.